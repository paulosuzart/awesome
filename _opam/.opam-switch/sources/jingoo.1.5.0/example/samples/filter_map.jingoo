[ {% for x in persons | map (null, attribute="name") %}{{x}} {% endfor %}]
[ {% for x in persons | map (attr ("extra.age")) %}{{x}} {% endfor %}]
