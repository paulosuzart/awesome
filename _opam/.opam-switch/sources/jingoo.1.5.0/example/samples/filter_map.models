{{ { persons : [ { name : "alice", extra: { age: 42, sex : "F" } },
                 { name : "bob", extra: { age: 34, sex : "M" } },
                 { name : "carol", extra: { age: 29, sex : "F" } } ] }
}}
