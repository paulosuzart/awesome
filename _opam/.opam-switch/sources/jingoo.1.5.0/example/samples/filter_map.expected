[ alice bob carol ]
[ 42 34 29 ]
